node_package/src/ReactOnRails.js

```diff
@@ -31,7 +31,7 @@ ctx.ReactOnRails = {
    */
   render(name, props, domNodeId) {
     const reactElement = createReactElement({ name, props, domNodeId });
-    ReactDOM.render(reactElement, document.getElementById(domNodeId));
+    return ReactDOM.render(reactElement, document.getElementById(domNodeId));
```

Comment:
We need to fix the JS Doc, and we should have a link to a file in the /docs directory that explains why you'd do this.